x/bridge/keeper/evm_withdraw_hooks_test.go

```diff
@@ -170,11 +174,74 @@ func (suite *EVMHooksTestSuite) TestERC20Withdraw_BalanceChange() {
 	)
 }
 
+func (suite *EVMHooksTestSuite) TestERC20Withdraw_MinimumWithdraw() {
+	toKey, err := ethsecp256k1.GenerateKey()
+	suite.Require().NoError(err)
+	withdrawToAddr := common.BytesToAddress(toKey.PubKey().Address())
+	withdrawAmount := testutil.MinWETHWithdrawAmount
+
+	// Send Withdraw TX
+	res := suite.Withdraw(suite.pair.GetInternalAddress(), withdrawToAddr, withdrawAmount.BigInt())
+
+	suite.Require().False(res.Failed(), "minimum withdraw should not fail")
```

Comment:
👍 Great to cover both the equals and less than cases here.  Great to be explicit on these reqs